source/draft_api.rst

```diff
@@ -223,7 +258,9 @@ Response body:
       "issue_uri":"https://github.com/osu-cass/whats-fresh-api/issues/56",
       "date_worked":null,
       "created_at":2014-09-18,
-      "updated_at":null
```

Comment:
Is this the request body, or the response? If request, why are we sending updated, deleted, and parent? Shouldn't the back end generate these?